### ISSUE TITLE
Azjol-Nerub 1-2 second boss alot of bugs ^^

### ISSUE THREAD

**Issue by itsflexlol** [dalaranwow/dalaran-wow#5608]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
Hi guys, ive noticed some bugs in you're Azjol-Nerub actually pretty many bugs in that 1 dungeon not sure why.
The very first two pulls you make in the dungeon are 100% correct but when you come to the first boss the bugs allready starts ill try to explain it as good as i can! 
So when you start on first boss this is how we where suposed to do it back in wotlk, you pull the first wave and when the first wave are killed the second wave will storm you and after that the 3rd then boss is suposed to come.
in your private server if you pull the first wave second wave allready comming when 1 add is dead ^^ so your left with 5 adds then you kill 1 more and 3rd wave comes so you have to deal with 7 adds at once then when 1 more dead you gotta deal with 6 enraged adds while boss is there. (this is wrong)
another bug is when you outrange the guards and pull the boss then the guards dont even help the boss lol.
that means the only way to do this boss is bugging the bug witch requires you got one in your group with a 40 yards spell^^ this is hopefully somthing you will be fixing lol cuse its almost impossible to do this dungeon at lvl 70+ cuse of that bug on first boss.
now time for the second boss i really wish i had a screen shot here tho cuse the very first guard is placed wrong lol, its suposed to stand and guard the middle infront of the stairs where all the small spiders are comming down from, when you killed that 1 add there will spawn 2 guards when they are dead the boss should move up kill the small spiders and we should be good to go.
now time for the bugs as i said 1 of the adds are placed wrong but that aint really an issue tho the issue is when you die at this boss all the guards will spawn none stop and you have no way what so ever to get the boss killed the 3 guards will keep on spawning and you will have to do all the 3 guards and the boss at once ^^

just to make things clear that i dont just trash talk, i have played world of warcraft for about 15 years, back in wotlk i was the very first person on EU region reaching lvl 80 with a human alliance affliction warlock named flexir, i topped the ladder boards in wotlk all seasons my best where at 3998 rating i really know what i am talking about and ive noticed SOOOO many bugs on this server its unreal, i am very willing to make alot of submits like this to help out if i can see that you guys are thankfull for it and actually fixing the issue, on top of that i wanna give you guys a MAJOR credit on how good the server actually are super fast loading screens and no lags thats really impressive keep it up! 
Regards Flexir Veteran world of warcraft gamer :)

**Comment by itsflexlol** [comment]
Hi again i actually found a perfect example on how it should work this link here is for a youtube guide from back in the days sadly this is heroic but you can see that on the first boss that i am right, they even skipped the guards on second boss because they are not suposed to respawn at all but rest fits perfectly in with what ive been explaining https://www.youtube.com/watch?v=ewJEd-L7Mg0 :)

**Comment by itsflexlol** [comment]
you should watch the entire vid this is 100% original how it should be even after second boss when he jumps down all the small spiders comming at once thats also a bug on you´re server that dosent happends hehe :) perfect example on how this dungeon is it really is guys.

**Comment by Lothloryen** [comment]
#3280

**Comment by itsflexlol** [comment]
Since this is closed now is it finally fixed ?

________________________________
Fra: Lothloryen <notifications@github.com>
Sendt: 9. maj 2019 15:23
Til: dalaranwow/dalaran-wow
Cc: itsflexlol; Author
Emne: Re: [dalaranwow/dalaran-wow] Azjol-Nerub 1-2 second boss alot of bugs ^^ (#5608)


Closed #5608<https://github.com/dalaranwow/dalaran-wow/issues/5608>.

**Comment by itsflexlol** [comment]
no fixed and closed wtf guys

**Comment by Chenocide** [comment]
This report is a duplicate. Go to #3280 and report any shortcomings there.
